js/deer-config.js

```diff
@@ -161,7 +161,12 @@ export default {
                 </style>
                 <h2> Glosses </h2>
                 <small class="cachedNotice is-hidden text-primary"> These Glosses were cached.  To reload the data <a class="newcache tag is-small">click here</a>. </small>
-                <input filter="title" type="text" placeholder="&hellip;Type to filter by incipit, text, or targeted text" class="is-hidden serifText">
+				<form id="ngForm" deer-type="named-gloss" deer-context="http://www.loc.gov/mods" class="row">
```

Comment:
The value for the deer-type attribute should be "Gloss".  "named-gloss" is the old type and we have since changed it.